### DISCUSSION TITLE
frontmatter is not a function

### DISCUSSION THREAD

**Discussion by shedali** [micromark/micromark#67] [Q&A] [answered]
testing out [micromark-extension-frontmatter](https://github.com/micromark/micromark-extension-frontmatter#options), following the readme on the repo

When I first run it, I get the error:
```
TypeError: micromark_extension_frontmatter_1.frontmatter is not a function
```

Repro here: https://stackblitz.com/edit/node-rp4sym?file=index.js

**Comment by wooorm** [comment, marked as answer]
Readme and code was recently updated for ESM, which is shipping soon, but not yet released (first want to do some more changes in micromark).
Assuming you can use ESM without problems, then you should be able to for now use:

```js
import frontmatter from 'micromark-extension-frontmatter'
```

In a couple of days, after updating everything, it would be as what’s now in the readme!

**Reply by wooorm** [reply]
The [previous readme](https://github.com/micromark/micromark-extension-frontmatter/blob/d83d5ea59a2447d54c893ec889e15fb848278bf5/readme.md) documents the currently published code. The current code is still requirable, the next code will have to be imported

**Reply by shedali** [reply]
just confirmed that works, thanks :)
